pallets/permissions/src/mock.rs

```diff
@@ -116,11 +116,12 @@ impl Properties for Storage {
 					self.org.remove(OrgStorage::HEAD_OF_SAUBERMACHING)
 				}
 			},
-		})
+		};
+		Ok(())
```

Comment:
Why this change?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#unit_arg